packages/braintree-utils/src/braintree-script-loader.ts

```diff
@@ -153,4 +177,106 @@ export default class BraintreeScriptLoader {
         // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
         return braintreeModule as T;
     }
+
+    // Info: for generating an integrity hash, you must use an appropriate hash generator https://www.srihash.org/
+    private getIntegrityValuesByFileName(fileName: string): BraintreeIntegrityValues {
```

Comment:
Instead of passing file name we can stick to module, therefore we don't need to have BraintreeSdkFileName enum at all